R/GCPC.R

```diff
@@ -199,8 +199,13 @@ write("GENETIC PREDICTIONS...", stderr())
 # in the matrix D before adding it to the estimated dominance marker effects
 # One unit of change in f represents changing all loci from homozygous to heterozygous
 
+if(userPloidy == 2){
 GC <- G - (userPloidy/2) #this centers G
 f <- rowSums(GC, na.rm = TRUE) / apply(GC, 1, function(x) sum(!is.na(x)))
+}
+if(userPloidy > 2){
+  GC <- G * (userPloidy - G) * (2 / userPloidy)^2
```

Comment:
what are the two encodings that we are converting here... this is not clear to me :-O